NuRadioMC/examples/06_webinar/W02RunSimulation.py

```diff
@@ -84,16 +84,15 @@ def _detector_simulation(self):
         Our simulation uses the default sampling rate of 5 GHz, or 5 GS/s, or
         equivalently, a time step of 0.2 ns. Such a high resolution, while needed
         during simulations to capture all the details of the radio wave, is not common
-        at all in radio experiments, where the sampling rates tend to lie around
-        the gigahertz. If we want our simulation to be a theoretical study, we
-        can use 5 GHz as the electric field sampling rate. However, if we want
-        to simulate what an actual radio detector would see,
-        we must resample to lower sampling rates, to the actual sampling rate of
-        our analog-to-digital converter. We have specified in our detector.json
-        file an adc_sampling_frequency of 2 GHz, which can be accessed using the
-        property _sampling_rate_detector.
+        at all in radio experiments after the wave has been digitised, with
+        sampling rates around the gigahertz. However, we are going to suppose
+        that our trigger is analog, so it sees a continuous waveform. We can upsample
+        our trace from 5 GS/s to 10 GS/s to illustrate the use of the channel
+        resampler. Keep in mind that if the trigger were digital and we needed
+        a sampling rate of, for instance, 2 GHz, the same channelResampler
+        module can also perform a downsampling.
         """
-        new_sampling_rate = self._sampling_rate_detector
+        new_sampling_rate = 2 * self._sampling_rate_detector
```

Comment:
sorry to bother again. But I think we should either use the `_sampling_rate_detector` if this is really the correct choice, or use the `internal sampling rate` from the config file i.e. `1/self._dt`. Otherwise this config property is useless.

Reply:
I don't agree. Using 1/self._dt to resample is the same as doing nothing, isn't it? The field is already sampled at 1/self._dt

Reply:
I don't mind using the _sampling_rate_detector. I just thought you didn't want to. Anyway, might as well remove all the resampling business if it's so controversial.

Reply:
no its not the same. The efieldtovoltageconverter upsamples the traces to 20GHz to have a good time precision when adjusting the efield for time delays. So downsampling afterwards is recommended for computing speed.

Reply:
I've introduced a comment on how to downsample if it's necessary. See if you're ok with that.